airbyte-integrations/connectors/source-jdbc/src/testFixtures/java/io/airbyte/integrations/source/jdbc/test/JdbcSourceStandardTest.java

```diff
@@ -83,18 +85,31 @@
 // 4. Then implement the abstract methods documented below.
 public abstract class JdbcSourceStandardTest {
 
-  private static final String SCHEMA_NAME = "jdbc_integration_test";
-  private static final String SCHEMA_NAME2 = "jdbc_integration_test2";
-  private static final Set<String> TEST_SCHEMAS = ImmutableSet.of(SCHEMA_NAME, SCHEMA_NAME2);
-
-  private static final String TABLE_NAME = "id_and_name";
-  private static final String TABLE_NAME_WITHOUT_PK = "id_and_name_without_pk";
-  private static final String TABLE_NAME_COMPOSITE_PK = "full_name_composite_pk";
-
-  private JsonNode config;
-  private JdbcDatabase database;
-  private AbstractJdbcSource source;
-  private static String streamName;
+  public String SCHEMA_NAME = "jdbc_integration_test";
+  public String SCHEMA_NAME2 = "jdbc_integration_test2";
+  public Set<String> TEST_SCHEMAS = ImmutableSet.of(SCHEMA_NAME, SCHEMA_NAME2);
+
+  public String TABLE_NAME = "id_and_name";
+  public String TABLE_NAME_WITH_SPACES = "id and name2";
+  public String TABLE_NAME_WITHOUT_PK = "id_and_name_without_pk";
+  public String TABLE_NAME_COMPOSITE_PK = "full_name_composite_pk";
+
+  public JsonNode config;
+  public JdbcDatabase database;
+  public AbstractJdbcSource source;
+  public static String streamName;
+
+  public String COL_ID = "id";
+  public static String COL_NAME = "name";
+  public static String COL_UPDATED_AT = "updated_at";
+  public String COL_FIRST_NAME = "first_name";
+  public String COL_LAST_NAME = "last_name";
+  public String COL_LAST_NAME_WITH_SPACE = "last name";
+  public Number COL_ID_VALUE_1 = 1;
+  public Number COL_ID_VALUE_2 = 2;
+  public Number COL_ID_VALUE_3 = 3;
+  public Number COL_ID_VALUE_4 = 4;
+  public Number COL_ID_VALUE_5 = 5;
```

Comment:
@sherifnada instead of overriding the function, I replace the hard-coded values with variables. Then, I just replace them in the Oracle class.